mantidproject/mantid
# Display save directory in statusbar of Eng Diff UI

Display save directory in the status bar of the Engineering diffraction UI (bottom of window)

**To test:**

(1) Interfaces > Engineering Diffraction
(2) Click the cog button at the bottom left of the window to open settings
(3) Check the save directory in the settings matches that in the status bar (bottom of UI window)
(4) Browse to a different save directory 
(5) Apply or save
(6) Check the save directory in the status bar has updated correctly

The save directory box has a max length, if the length of the directory string exceeds this you should still be able to view the full path by hovering over the mouse over it to display the tool tip.

Fixes #29063 

*This does not require release notes* because **very minor change to info displayed**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
